internal/admin/dashboard/static/css/dashboard.css

```
@@ -0,0 +1,420 @@
+        --chart-text: #7a7068;
+        --chart-tooltip-bg: #ffffff;
+        --chart-tooltip-border: #e8e0d6;
+        --chart-tooltip-text: #2d2519;
+        color-scheme: light;
+    }
+}
+
+* { box-sizing: border-box; margin: 0; padding: 0; }
+
+body {
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)